src/main/java/org/fungover/thunder/Broker.java

```diff
@@ -4,13 +4,13 @@
 import java.net.ServerSocket;
 import java.net.Socket;
 
-public class Server {
+public class Broker {
```

Comment:
We could make this its own PR or are we sure other coders can handle this seamless so we don't end up with two classes, both a Server and a Broker class? Is this really a Broker and is it inside the scope of the PR?

Reply:
There was not much to change as I did the PR, but now it could be more problematic to make this change. So probably I should dismiss my changes. I have lately also thougt that is could as well be called Server :-)